### PR TITLE
Polaris 4.6.0 Change log update

### PR DESCRIPTION
**Pull request by dleroux**
Polaris 4.6.0 Change log update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 4.6.0 - 2019-16-08
+
+### Enhancements
+
+- Added `totalItemsCount` prop to `ResourceList` component ([2233](https://github.com/Shopify/polaris-react/pull/2233))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Added a `totalItemsCount` prop to the `ResourceList` component ([2233](https://github.com/Shopify/polaris-react/pull/2233))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Prevented the `Header` primary action label on `Page` from wrapping when the title is too long ([2262](https://github.com/Shopify/polaris-react/pull/2262))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Prevented the `Header` primary action label on the `Page` component from wrapping when the title is too long ([2262](https://github.com/Shopify/polaris-react/pull/2262))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+
+### Bug fixes
+
+- Fixed issue with `Stack` where a `Stack.Item` was not getting a minimum width ([2273](https://github.com/Shopify/polaris-react/pull/2273))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Fixed an issue with the `Stack` component where a `Stack.Item` was not getting a minimum width ([2273](https://github.com/Shopify/polaris-react/pull/2273))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+- Fixed issue with `Filters` applying inconsistent border styles to sibling filters and when
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Fixed an issue with `Filters` applying inconsistent border styles to sibling filters when
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+  there is only one filter in the filter list ([2284](https://github.com/Shopify/polaris-react/pull/2284))
+- Added `aria-disabled` to select component content when it is disabled ([#2281](https://github.com/Shopify/polaris-react/pull/2281))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Added `aria-disabled` to the select component's content when it is disabled ([#2281](https://github.com/Shopify/polaris-react/pull/2281))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+
+### Documentation
+
+- Added accessibility documentation for the `DropZone` component ([#2243](https://github.com/Shopify/polaris-react/pull/2243))
+- Improved accessibility documentation for the `Spinner` component ([#2258](https://github.com/Shopify/polaris-react/pull/2258))
+
+### Development workflow
+
+- Add support for context customization in Storybook using addon-contexts ([#2281](https://github.com/Shopify/polaris-react/pull/2281))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Added support for context customization in Storybook using addon-contexts ([#2281](https://github.com/Shopify/polaris-react/pull/2281))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
...
+
+### Code quality
+
+- Migrated `DateSelector` to use hooks instead of withAppProvider ([#2193](https://github.com/Shopify/polaris-react/pull/2193))
+- Migrated `Toast` to use hooks ([#2222](https://github.com/Shopify/polaris-react/pull/2222))
+- Removed `link`, `theme` and `scrollLockManager` from the object returned by withAppProvider as nothing consumes them any more ([#2277](https://github.com/Shopify/polaris-react/pull/2277))
```
Comment (danrosenthal) [inline CHANGELOG.md]:
```suggestion
- Removed `link`, `theme` and `scrollLockManager` from the object returned by `withAppProvider` as nothing consumes them any more ([#2277](https://github.com/Shopify/polaris-react/pull/2277))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -8,6 +8,35 @@ The format is based on [these versioning and changelog guidelines](https://git.i
 
 ---
 
+## 4.6.0 - 2019-16-08
+
+### Enhancements
+
+- Added a `totalItemsCount` prop to the `ResourceList` component ([2233](https://github.com/Shopify/polaris-react/pull/2233))
+- Prevented the `Header` primary action label on the `Page` component from wrapping when the title is too long ([2262](https://github.com/Shopify/polaris-react/pull/2262))
```
Comment (tmlayton) [inline CHANGELOG.md]: This seems like a bug fix?

**Pull request merged**